behat/tests/bootstrap/v2/common/IShouldBeOnTrait.php

```diff
@@ -13,61 +13,71 @@ public function iAmOnPage(string $urlRegex)
         if (!$onExpectedPage) {
             $this->throwContextualException(sprintf('Not on expected page. Current URL is: %s', $currentUrl));
         }
+
+        return true;
     }
 
     /**
      * @Then I should be on the report submitted page
      */
     public function iAmOnReportSubmittedPage()
     {
-        $this->iAmOnPage('/report\/.*\/submitted$/');
+        return $this->iAmOnPage('/report\/.*\/submitted$/');
     }
 
     /**
      * @Then I should be on the post-submission user research page
      */
     public function iAmOnPostSubmissionUserResearchPage()
     {
-        $this->iAmOnPage('/report\/.*\/post_submission_user_research$/');
+        return $this->iAmOnPage('/report\/.*\/post_submission_user_research$/');
     }
 
     /**
      * @Then I should be on the user research feedback submitted page
      */
     public function iAmOnUserResearchSubmittedPage()
     {
-        $this->iAmOnPage('/report\/.*\/post_submission_user_research\/submitted$/');
+        return $this->iAmOnPage('/report\/.*\/post_submission_user_research\/submitted$/');
     }
 
     /**
      * @Then I should be on the contacts summary page
      */
     public function iAmOnContactsSummaryPage()
     {
-        $this->iAmOnPage('/report\/.*\/contacts\/summary$/');
+        return $this->iAmOnPage('/report\/.*\/contacts\/summary$/');
     }
 
     /**
      * @Then I should be on the add a contact page
      */
     public function iAmOnAddAContactPage()
     {
-        $this->iAmOnPage('/report\/.*\/contacts\/add/');
+        return $this->iAmOnPage('/report\/.*\/contacts\/add/');
     }
 
     /**
      * @Then I should be on the contacts add another page
      */
     public function iAmOnContactsAddAnotherPage()
     {
-        $this->iAmOnPage('/report\/.*\/contacts\/add_another$/');
+        return $this->iAmOnPage('/report\/.*\/contacts\/add_another$/');
+    }
+
+    /**
+     * @Then I should be on the additional information summary page
+     */
+    public function iAmOnAdditionalInformationSummaryPage()
+    {
+        return $this->iAmOnPage('/report\/.*\/any-other-info\/summary\?from=last-step/');
```

Comment:
Not sure what impact it has on the page but we could probably drop the query param here and just match on the URI:

```
/report\/.*\/any-other-info\/summary/
```